composer-lagoon.json

```diff
@@ -27,7 +27,7 @@
         "drupal/console": "^1.0.2",
         "drupal/core": "8.6.7",
         "drush/drush": "^9.0.0",
-        "govcms/govcms": "1.0.0-beta1",
+        "govcms/govcms": "*",
```

Comment:
@tobybellwood With your latest commit adding "govcms:govcms: *" in composer-lagoon.json, the build ends up with a docroot directory under profiles/contrib/govcms and it results in installation failure.
```
PHP Fatal error:  Cannot redeclare twig_theme() (previously declared in /app/web/profiles/contrib/govcms/docroot/core/themes/engines/twig/twig.engine:15) in /app/web/core/themes/engines/twig/twig.engine on line 19
 [error]  Drush command terminated abnormally due to an unrecoverable error.
Error: Cannot redeclare twig_theme() (previously declared in /app/web/profiles/contrib/govcms/docroot/core/themes/engines/twig/twig.engine:15) in /app/web/core/themes/engines/twig/twig.engine, line 19
```
```
$ ahoy cli

cli-drupal:/app$ grep -r 'function twig_theme(' web
web/profiles/contrib/govcms/docroot/core/themes/engines/twig/twig.engine:function twig_theme($existing, $type, $theme, $path) {
web/core/themes/engines/twig/twig.engine:function twig_theme($existing, $type, $theme, $path) {
```